packages/web/src/scss/components/Grid/_tools.scss

```diff
@@ -26,7 +27,7 @@
     @each $breakpoint-name, $breakpoint-value in $breakpoints {
```

Comment:
Off-topic: the name of the first mixin argument (line 25 is out of the scope of this PR) could be just `$column-count`. It's already stated in the mixin name the columns will be equal(-width).